<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>1200</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>1200</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1200</width>
    <height>1200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1171</width>
      <height>1151</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="16" column="0">
      <widget class="QTextBrowser" name="commands">
       <property name="styleSheet">
        <string notr="true">font: 75 10pt &quot;맑은 고딕&quot;;</string>
       </property>
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'맑은 고딕'; font-size:10pt; font-weight:72; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Gulim'; font-size:9pt; font-weight:400;&quot;&gt;Not Started Yet..&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
      <widget class="QLabel" name="label_3">
       <property name="styleSheet">
        <string notr="true">font: 75 20pt &quot;맑은 고딕&quot;;</string>
       </property>
       <property name="text">
        <string>Interface List</string>
       </property>
      </widget>
     </item>
     <item row="11" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>4</number>
       </property>
       <item alignment="Qt::AlignRight">
        <widget class="QLabel" name="label">
         <property name="styleSheet">
          <string notr="true">font: 75 14pt &quot;맑은 고딕&quot;;</string>
         </property>
         <property name="text">
          <string>Managed</string>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignLeft">
        <widget class="QSpinBox" name="managed_Num"/>
       </item>
       <item alignment="Qt::AlignRight">
        <widget class="QLabel" name="label_2">
         <property name="styleSheet">
          <string notr="true">font: 75 14pt &quot;맑은 고딕&quot;;</string>
         </property>
         <property name="text">
          <string>Monitor</string>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignLeft">
        <widget class="QSpinBox" name="monitor_Num"/>
       </item>
       <item>
        <widget class="QPushButton" name="setInterfaceButton">
         <property name="minimumSize">
          <size>
           <width>500</width>
           <height>3</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 75 20pt &quot;맑은 고딕&quot;;</string>
         </property>
         <property name="text">
          <string>Set Interface</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="9" column="0">
      <widget class="QTableWidget" name="NetworkList">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="sizeAdjustPolicy">
        <enum>QAbstractScrollArea::AdjustToContents</enum>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <attribute name="horizontalHeaderVisible">
        <bool>true</bool>
       </attribute>
       <attribute name="horizontalHeaderCascadingSectionResizes">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>250</number>
       </attribute>
       <attribute name="horizontalHeaderMinimumSectionSize">
        <number>40</number>
       </attribute>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <column>
        <property name="text">
         <string>No</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Name</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Description</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>MAC</string>
        </property>
       </column>
      </widget>
     </item>
     <item row="17" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QComboBox" name="command">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 75 10pt &quot;맑은 고딕&quot;;</string>
         </property>
         <item>
          <property name="text">
           <string>Reset network interface to use</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>List nearby WiFi networks</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Collect IP and MAC addresses of all hosts(AP and client)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>(not perfect)Send deauth frames(force disconnect the target client from the network)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>(not yet supported)Create a Rogue AP</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>(not yet supported)Test KRACK - 4-way handshake reinstall PTK-TK(used to encrypt data frames)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>(not yet supported)Test KRACK - 4-way handshake reinstall GTK(used to encrypt broadcast and multicast frames)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>(not yet supported)Test KRACK - 4-way handshake reinstall IGTK(used to encrypt broadcast and multicast frames)</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="runButton">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>0</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 75 20pt &quot;맑은 고딕&quot;;</string>
         </property>
         <property name="text">
          <string>Run</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
